apps/desktop/desktop/src/main/java/bisq/desktop/main/content/settings/preferences/PreferencesView.java

```diff
@@ -277,6 +276,18 @@ protected void onViewAttached() {
         getSelectedSupportedLanguageCodePin = EasyBind.subscribe(model.getSelectedLSupportedLanguageCode(),
                 e -> supportedLanguagesComboBox.getSelectionModel().select(e));
 
+        ignoreDiffAdjustFromSecManagerSwitchPin = EasyBind.subscribe(
+                ignoreDiffAdjustFromSecManagerSwitch.selectedProperty(), isSelected -> {
+           model.getIgnoreDiffAdjustmentFromSecManager().set(isSelected);
+           difficultyAdjustmentFactor.validate();
+        });
+
+        ignoreMinRequiredReputationScoreFromSecManagerSwitchPin = EasyBind.subscribe(
+                ignoreMinRequiredReputationScoreFromSecManagerSwitch.selectedProperty(), isSelected -> {
+            model.getIgnoreMinRequiredReputationScoreFromSecManager().set(isSelected);
+            minRequiredReputationScore.validate();
+        });
+
```

Comment:
Is the update from the model to the switch handled? It was bi-directional binding before not its like a normal binding of switch to model.
I guess we only need it at active to apply the model date to the switch as the model data gets not changed by any other source.

Reply:
Good catch. No need to remove it then, let's just use the listener to trigger validation.